lib/librte_hash/rte_cuckoo_hash.c

```diff
@@ -1160,3 +1160,40 @@ rte_hash_iterate(const struct rte_hash *h, const void **key, void **data, uint32
 
 	return position - 1;
 }
+
+int32_t __rte_experimental
+rte_hash_iterate_conflict_entries_with_hash(const struct rte_hash *h,
+	const void **key, const void **data, hash_sig_t sig, uint32_t *next)
+{
+	RETURN_IF_TRUE(((h == NULL) || (next == NULL)), -EINVAL);
```

Comment:
Parameters `key` and `data` are not optional, so test for them in `RETURN_IF_TRUE()`.